*** start_pms_old	2012-06-30 12:06:14.000000000 +0200
--- start_pms	2012-06-30 12:08:33.000000000 +0200
***************
*** 7,13 ****
  export PLEX_MEDIA_SERVER_TMPDIR=/tmp
  export PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR="${HOME}/Library/Application Support"
  
! test -f /etc/default/plexmediaserver && . /etc/default/plexmediaserver
  
  export LD_LIBRARY_PATH="${PLEX_MEDIA_SERVER_HOME}"
  export TMPDIR="${PLEX_MEDIA_SERVER_TMPDIR}"
--- 7,13 ----
  export PLEX_MEDIA_SERVER_TMPDIR=/tmp
  export PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR="${HOME}/Library/Application Support"
  
! test -f /etc/plex/plexmediaserver.conf && . /etc/plex/plexmediaserver.conf
  
  export LD_LIBRARY_PATH="${PLEX_MEDIA_SERVER_HOME}"
  export TMPDIR="${PLEX_MEDIA_SERVER_TMPDIR}"
