diff --git a/meld/meldbuffer.py b/meld/meldbuffer.py
index 1ba2bb2..2fc8507 100644
--- a/meld/meldbuffer.py
+++ b/meld/meldbuffer.py
@@ -42,6 +42,11 @@ class MeldBuffer(GtkSource.Buffer):
         self.data = MeldBufferData(filename)
         self.user_action_count = 0
 
+        # Syntax highlighting scheme selection
+        manager = GtkSource.StyleSchemeManager.get_default()
+        style = GtkSource.StyleSchemeManager.get_scheme(manager, 'mrpdaemon')
+        GtkSource.Buffer.set_style_scheme(self, style)
+
     def do_begin_user_action(self, *args):
         self.user_action_count += 1
 
